proposals/lip-0052.md

```diff
@@ -2295,7 +2295,23 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
 
 ### Endpoints for Off-Chain Services
 
-This section specifies the non-trivial or recommended endpoints of the NFT module and does not include all endpoints. TBA
+This section specifies the non-trivial or recommended endpoints of the NFT module and does not include all endpoints. 
+
+#### hasNFT
+
+This function checks whether an NFT is owned by a specific user.
+
+#### getNFT
+
+This function returns NFT substore information for a provided NFT ID.
```

Comment:
The implementation also returns the locking module from the user substore. I would modify the sentence to:
"This function returns NFT and user substore information for a provided NFT ID."

Reply:
Done.